prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -371,6 +406,9 @@ class FHIRRouter(
         filter: ReportStreamFilter?,
         bundle: Bundle,
         defaultResponse: Boolean,
+        report: Report,
```

Comment:
I don't think we want the `report` in this function. Tracking the results should be done outside of the 'evaluateFilterCondition' because you have now made this 'evaluateFilterConditionAndTrackResults'

Reply:
cleaned the parameters up on that function while taking out the logging from it